conf/log4j.properties

```diff
@@ -12,63 +12,121 @@
-log4j.appender.coverage=org.apache.log4j.FileAppender
-log4j.appender.coverage.File=${outputDirectory}/coverage.txt
-log4j.appender.coverage.append=false
-log4j.appender.coverage.layout=org.apache.log4j.PatternLayout
-log4j.appender.coverage.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
-
-log4j.logger.com.zimbra.qa.selenium.framework.util.SleepMetrics=INFO,sleep
-log4j.appender.sleep=org.apache.log4j.FileAppender
-log4j.appender.sleep.File=${outputDirectory}/sleep.txt
-log4j.appender.sleep.append=false
-log4j.appender.sleep.layout=org.apache.log4j.PatternLayout
-log4j.appender.sleep.layout.ConversionPattern=%-4r - %m%n
+appender.A3.layout.type=PatternLayout
+appender.A3.layout.pattern=m%n
+
+logger.A3.name = A3File
+logger.A3.level = info
+logger.A3.additivity = false
+logger.A3.appenderRef.stdout.ref = a3FileAppender
+
+appender.general.type=RollingFile
+appender.general.fileName=${outputDirectory}/debug.txt
+appender.general.name=generalFileAppender
+appender.general.level=WARN
+appender.general.append=false
+appender.general.layout=org.apache.log4j.PatternLayout
+appender.general.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
+
+logger.general.name = generalFileLogger
+logger.general.level = info
+logger.general.additivity = false
+logger.general.appenderRef.file.ref = generalFileAppender
+
+
+logger.A5.name = com.zimbra.qa.selenium.framework.util.staf.StafAbstract
+logger.A5.level = info
+logger.A5.additivity = false
+logger.A5.appenderRef.stdout.ref = A1
+logger.A5.appenderRef.file.ref = coverageFileAppender
+
+
+logger.A5.name = com.zimbra.qa.selenium.framework.util.CodeCoverage
+logger.A5.level = info
+logger.A5.additivity = false
+logger.A5.appenderRef.stdout.ref = A1
+logger.A5.appenderRef.file.ref = coverageFileAppender
+
+logger.A5.name = com.zimbra.qa.selenium.framework.core.ExecuteHarnessMain.MethodListener
+logger.A5.level = info
+logger.A5.additivity = false
+logger.A5.appenderRef.stdout.ref = A1
+logger.A5.appenderRef.file.ref = coverageFileAppender
+
+appender.coverage.type=RollingFile
+appender.coverage.fileName=${outputDirectory}/coverage.txt
+appender.coverage.name=coverageFileAppender
+appender.coverage.level=WARN
+appender.coverage.append=false
+appender.coverage.layout=org.apache.log4j.PatternLayout
+appender.coverage.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
+
+logger.coverage.name = converageFile
+logger.coverage.level = info
+logger.coverage.additivity = false
+logger.coverage.appenderRef.file.ref = coverageFileAppender
+
+logger.coverage.name = com.zimbra.qa.selenium.framework.util.SleepMetrics
+logger.coverage.level = info
+logger.coverage.additivity = false
+logger.coverage.appenderRef.file.ref = coverageFileAppender
+
+appender.sleep.type=RollingFile
+appender.sleep.fileName=${outputDirectory}/sleep.txt
+appender.sleep.level=INFO
+appender.sleep.append=false
+appender.sleep.layout=org.apache.log4j.PatternLayout
```

Comment:
Use PatternLayout only as org.apache.log4j.PatternLayout is belongs to log4j-1.2.16,